source/predicates/string.ts

```diff
@@ -58,6 +58,27 @@ export class StringPredicate extends Predicate<string> {
 		});
 	}
 
+	/**
+	Test a string against a character set.
+
```

Comment:
You need to mention that `ow.characterSet` comes with some predefined character sets.